ETC.tsv

```diff
@@ -1745,7 +1745,7 @@ ETC_20150317_001744	Not possible on this item.
 ETC_20150317_001745	This item is not equipped with a gem.
 ETC_20150317_001746	Please select a socket.
 ETC_20150317_001747	Show all
-ETC_20150317_001748	Consume
+ETC_20150317_001748	Consumable
```

Comment:
Line 6176 shows Consumable as the category for items in game, so this one should stay as "Consume".